quantecon/game_theory/normal_form_game.py

```diff
@@ -276,8 +276,7 @@ def best_response(self, opponents_actions, tie_breaking='smallest',
 
         Parameters
         ----------
-        opponents_actions : array_like(int or array_like(float)) or
-                            array_like(int, ndim=1) or scalar(int)
+        opponents_actions : scalar(int) or array_like
```

Comment:
Thank you for comments! I am wondering whether `array_like(int or float)` could cover `array_like(array_like(float))`? If this is the case, `array_like(int or float)` would be better.